content/concepts/targets.md

```diff
@@ -1,10 +1,80 @@
 ---
 title: Targets
 ---
-> webworker
-> node
-> async-node
-> node-webkit
-> electron
-> electron-main
-> electron-renderer
+## Overview
+
+Because JavaScript can be written for both server & browser, *webpack* offer's multiple deployment _targets_.
```

Comment:
* Better as "server and browser".
* /s/offer's/offers
* Maybe stick to just webpack everywhere so that formatting is consistent.